src/Clock.tsx

```diff
@@ -0,0 +1,37 @@
+/* eslint-disable no-console */
+import React, { useState, useEffect } from 'react';
+
+type Props = {
+  name: string;
+};
+
+export const Clock: React.FC<Props> = ({ name }) => {
+  const [date, setDate] = useState(new Date());
+
+  useEffect(() => {
+    const timerId = window.setInterval(() => {
+      setDate(new Date());
+      console.info(date.toUTCString().slice(-12, -4));
```

Comment:
This is logging the same time always because it's reading date from the initial state. Try creating a variable with a value new Date() and use it for setDate and console.info